carma_ros2_utils/CMakeLists.txt

```diff
@@ -5,50 +5,46 @@ find_package(carma_cmake_common REQUIRED)
 carma_check_ros_version(2)
 carma_package()
```

Comment:
Is this needed?

Reply:
Yes all carma_packages should have this it sets the default c++ version and a few other build flags